org.scala-ide.sdt.core/src/org/scalaide/extensions/saveactions/RemoveDuplicatedEmptyLines.scala

```diff
@@ -34,6 +34,9 @@ trait RemoveDuplicatedEmptyLines extends SaveAction with DocumentSupport {
           Seq()
     }
 
-    removedLines.toList
+    if (emptyLines.length == 1)
```

Comment:
It's not obvious to me if `emptyLines` is a linear sequence (thus `length` can be slow), or not. If not, LGTM :)